zh/option/partial/strictly-aligned.md

```diff
@@ -0,0 +1,9 @@
+{{ target: partial-strictly-aligned }}
+
+## strictlyAligned(boolean) = false
+
+强制线条对齐，在笛卡尔坐标系([coordinateSystem](~series-heatmap.coordinateSystem): 'cartesian2d')上有效。
+
+**注意**：此配置生效后，在数据量很大时可能会有性能问题。
```

Comment:
Hi, can you put an example of how this can cause performance issues?

Reply:
I remember that CANVAS has an optimization to ensure that the number is an integer, So I came to the conclusion.

This is MDN docs.
[MDN](https://developer.mozilla.org/zh-CN/docs/Web/API/Canvas_API/Tutorial/Optimizing_canvas#%E9%81%BF%E5%85%8D%E6%B5%AE%E7%82%B9%E6%95%B0%E7%9A%84%E5%9D%90%E6%A0%87%E7%82%B9%EF%BC%8C%E7%94%A8%E6%95%B4%E6%95%B0%E5%8F%96%E8%80%8C%E4%BB%A3%E4%B9%8B)

Reply:
i create a [DEMO](https://github.com/xiaoyueguang/echarts-heatmap-demo).

After the test, the performance is not bad.

So do i remove this notice? @pissang

Reply:
I'm not sure. Performance optimization can be outdated. If the benchmark shows the performance is similar, we can remove this notice to avoid confusion.